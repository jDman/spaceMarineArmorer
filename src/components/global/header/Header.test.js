import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

test('renders Header component', () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
