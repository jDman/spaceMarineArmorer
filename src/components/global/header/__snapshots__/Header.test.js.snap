// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header component 1`] = `
<header>
  <h1>
    Space Soldier Armory
  </h1>
</header>
`;
